packages/admin/src/form/RangeSlider.tsx

```diff
@@ -0,0 +1,131 @@
+import { FormControl, Slider, SliderProps, Theme, WithStyles } from "@material-ui/core";
+import { createStyles, withStyles } from "@material-ui/styles";
+import * as React from "react";
+import { FieldRenderProps } from "react-final-form";
+
+import { InputBase } from "./InputBase";
+
+export type CometAdminRangeSliderClassKeys = "root" | "inputsWrapper" | "inputFieldsSeperatorContainer" | "sliderWrapper" | "inputFieldContainer";
+
+const styles = (theme: Theme) => {
+    return createStyles<CometAdminRangeSliderClassKeys, any>({
+        root: {
+            padding: "0 20px",
+            width: "100%",
+        },
+        inputsWrapper: {
+            justifyContent: "space-between",
+            marginBottom: "15px",
+            alignItems: "center",
+            display: "flex",
+        },
+        inputFieldsSeperatorContainer: {
+            textAlign: "center",
+            minWidth: "20%",
+        },
+        sliderWrapper: {
+            paddingBottom: "20px",
+        },
+        inputFieldContainer: {
+            textAlign: "center",
+        },
+    });
+};
+
+interface IRangeSliderProps extends FieldRenderProps<any, HTMLInputElement> {
```

Comment:
This Input is a component to be used with FinalForm (it extends `FieldRenderProps`), the component name should therefore  be prefixed with `FinalForm` => `FinalFormRangeSlider`  (https://github.com/vivid-planet/comet-admin/issues/254)

see other components in `packages/admin/src/form`